MVP Architecture/Presenter/LoginPresenter.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  LoginPresenter.swift
+//  iOSPracticals
+//
+//  Created by Yagnik Bavishi on 12/04/22.
+//
+
+import Foundation
+import Alamofire
+
+protocol LoginSuccess {
+    func loginSuccessFul()
+    func loginUnSuccessFul()
```

Comment:
successful is one word so it should be like loginSuccessful and loginUnSuccessful